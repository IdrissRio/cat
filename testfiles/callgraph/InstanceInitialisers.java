// PackageName: org.extendj.callgraph.InstanceInitialisers
// MethodName: main
package org.extendj.callgraph;
class A extends T {
  public A() { System.out.println("A"); }
}

class T {
  {
    System.out.println("Initialiser");
  }
  public T() { System.out.println("T"); }
}

class InstanceInitialisers {
  public static void main(String[] args) {
    A a = new A();
    A ab = new A();
  }
}