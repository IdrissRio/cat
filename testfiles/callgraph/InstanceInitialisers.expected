A::A -> PrintStream::println, T::T
InstanceInitialisers::main -> A::A
Object::<init>
PrintStream::println
T::T -> Object::<init>, PrintStream::println, T::init<>
T::init<> -> PrintStream::println

